capstone-frontend/src/components/BookDescriptionOverlay.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Popover, OverlayTrigger } from 'react-bootstrap';
+
+const popoutDescription = (book) => {
+  let description = book.description;
```

Comment:
You could combine the assignment in lines 5-8 into one line using javascript short-circuit expressions. Check this out for guidance: https://codeburst.io/javascript-what-is-short-circuit-evaluation-ff22b2f5608c. The following assignment:
```
let description = book.description || "";
``` 
would have the same behavior as your current implementation and be a little cleaner.

Reply:
This is a good idea! I knew about the && but not using || in this case